.github/workflows/labels.yml

```diff
@@ -124,19 +124,11 @@ jobs:
                 },
               },
               issues: {
-                // Termux issues are usually user error. However they may describe issues within modules/framework itself,
-                // and for this reason they are not closed automatically.
+                // Termux issues should be nuked from orbit.
                 termux: {
-                  close: false,
+                  close: true,
                   comment: `
-                    Termux is not officially supported. https://github.com/rapid7/metasploit-framework/issues/11023
-
-                    However, Metasploit reportedly does work with Termux.
-
-                    Refer to the following for more information:
-
-                    * https://wiki.termux.com/wiki/Metasploit_Framework
-                    * termux/termux-packages/issues/715
+                    Metasploit installation on Termux is not supported. Termux does not support hacking tools. https://wiki.termux.com/wiki/Hacking
```

Comment:
Since we still have pointers to the original #11023 thread as being the source of truth, as well as it being referenced in the 'new issue' templates here, and probably other places:

https://github.com/rapid7/metasploit-framework/blob/c66f25c6585f830a6636d6cb25b891af8a5c0b90/.github/ISSUE_TEMPLATE/config.yml

What are your thoughts on updating the bot's response to be more generic:
```suggestion
                    Metasploit installation on Termux is not supported. Refer to the following for more information:
                    
                    * https://github.com/rapid7/metasploit-framework/issues/11023
```

And updating https://github.com/rapid7/metasploit-framework/issues/11023 to be:

```
Metasploit installation on Termux is not supported. Termux does not support hacking tools. 

* https://wiki.termux.com/wiki/FAQ#Where_are_Metasploit_and_Hydra_packages
* https://wiki.termux.com/wiki/Hacking
```

That way we can modify the latest messaging in one place, rather than having potentially stale bot comments in the future if the wording ever changes again

Reply:
I'm not fussed either way. Also not fussed if this is merged as is.

> And updating #11023 to be:

I've updated the issue.

Reply:
Thanks!